.ado/pipelines/templates/stages-locust.yaml

```diff
@@ -22,10 +22,10 @@ parameters:
 stages:
```

Comment:
I guess we leave the file name untouched for now and later might have two different stage files for ALT and Locust?

Reply:
Yes. The other one already exists. For now its just about the displayNames.